src/rpc/mining.cpp

```diff
@@ -815,7 +815,6 @@ UniValue estimatesmartfee(const JSONRPCRequest& request)
             "\n"
             "A negative value is returned if not enough transactions and blocks\n"
             "have been observed to make an estimate for any number of blocks.\n"
-            "However it will not return a value below the mempool reject fee.\n"
```

Comment:
I think we should at least flag the change in the release notes but I don't think that's necessarily something users need to worry about.  In the event that the tx isn't accepted to the mempool it's still likely'ish to be confirmed within the target by being resubmitted after mempool min fee decays.  Which makes me wonder if we should even be worrying about mempool min fee in the GetMinimumFee logic but enough changes for now.